### DISCUSSION TITLE
Bitbake Command not found

### DISCUSSION THREAD

**Discussion by maitreyakale** [polarfire-soc/polarfire-soc-documentation#312] [Linux build systems for PolarFire SoC]
We are trying to implement Yocto Project using Icicle Kit ! we followed all the steps that given in the https://github.com/polarfire-soc/meta-polarfire-soc-yocto-bsp#Dependencies ,but during implementation of "**MACHINE=icicle-kit-es bitbake mpfs-dev-cli**"  command following error i.e. "bitbake: command not found" is occured ! please give solution into this.....

**Comment by vfalanis** [comment]
Closing this discussion as this is a duplicate of discussion [#310](https://github.com/orgs/polarfire-soc/discussions/310).
